pwiz_tools/Skyline/TestData/XmlValidationTest.cs

```diff
@@ -58,7 +58,7 @@ public void TestCurrentXmlFormat()
             // Test schema validation.
             var assembly = Assembly.GetAssembly(typeof(AssertEx));
             var stream = assembly.GetManifestResourceStream(
-               typeof(AssertEx).Namespace + String.Format(CultureInfo.InvariantCulture, @".Schemas.Skyline_{0}.xsd", SrmDocument.FORMAT_VERSION));
+               typeof(AssertEx).Namespace + String.Format(CultureInfo.InvariantCulture, @".Schemas.Skyline_Current.xsd"));

```

Comment:
This should use the method:
SchemaDocuments.GetSkylineResourceName

Reply:
fixed, thanks